### PR TITLE
fix(UI): Styled the measurement control bar on the topology graphs per design

### PR DESCRIPTION
**Pull request by shazlehu**
### Proposed Change

This fixes the styling on the menu selecting Logs, Metrics, or Traces in the overview & configuration graphs. Also cleans up some of the naming of the .scss files.

##### Checklist

- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import { AppBar, Toolbar, Button, Typography, Box } from "@mui/material";
+import { AppBar, Toolbar, Button, Typography, Box, Stack } from "@mui/material";
+import { classes } from "../../utils/styles";
 
-import styles from "./styles.module.scss";
+import styles from "./measurement.bar.styles.module.scss";
```
Comment (dsvanlani) [inline ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx:4]: nit: I've been using snake-case for .scss files.  Can we rename this file to`measurement-control-bar.module.scss` ?

File: ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx [hunk 2]
```diff
@@ -39,25 +40,69 @@ export const MeasurementControlBar: React.FC<MeasurementControlBarProps> = ({
 }) => {
   return (
     <AppBar classes={{ root: styles.appbar }} color="primary" position="static">
-      <Toolbar>
+      <Toolbar classes={{ root: styles.toolbar }}>
```
Comment (dsvanlani) [inline ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx:43]:
```suggestion
      <Toolbar classes={{ root: styles.toolbar }} key={t}>
```

Comment (dsvanlani) [reply ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx:43]: image

File: ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx [hunk 3]
```diff
@@ -39,25 +40,69 @@ export const MeasurementControlBar: React.FC<MeasurementControlBarProps> = ({
...
-        {Object.entries(PERIODS).map(([r, label]) => (
-          <Button key={r} color="inherit" onClick={() => onPeriodChange(r)}>
-            <Typography style={{ fontWeight: period === r ? 700 : 300 }}>
-              {label}
-            </Typography>
-          </Button>
+        <Box className={styles["flex-box"]} />
+
+        {Object.entries(PERIODS).map(([p, label]) => (
+          <Stack>
```
Comment (dsvanlani) [inline ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx:78]:
```suggestion
          <Stack key={p}>
```

File: ui/src/components/MeasurementControlBar/measurement.bar.styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  border-color: colors.$transparent;
+  border: 0;
+}
+
+.menu-button-selected {
+  border-color: colors.$active-blue;
+}
+
+.regular-box {
+  border: 2px solid;
```
Comment (dsvanlani) [inline ui/src/components/MeasurementControlBar/measurement.bar.styles.module.scss:48]:
```suggestion
  border: 2px solid colors.$transparent;
```

File: ui/src/components/MeasurementControlBar/measurement.bar.styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  border-radius: 10px;
+  margin: 1px;
+  padding: 0px;
+  border-color: colors.$transparent;
```
Comment (dsvanlani) [inline ui/src/components/MeasurementControlBar/measurement.bar.styles.module.scss:52]:
```suggestion
```

File: ui/src/components/MeasurementControlBar/measurement.bar.styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  padding-bottom: 0;
+}
+
+.toolbar {
+  padding-left: 0;
+  padding-right: 0;
+}
+
+.flex-box {
+  //  border-bottom: 1px solid colors.$mui-transparent-gray;
```
Comment (dsvanlani) [inline ui/src/components/MeasurementControlBar/measurement.bar.styles.module.scss:16]:
```suggestion
```

File: ui/src/styles/_colors.scss [hunk 7]
```diff
@@ -28,3 +28,9 @@ $medium-gray: #d9d9d9;
 $gray: #e5e5e5;
 $light-gray: #f8f8f8;
 $white: #ffffff;
+
+// taken from the design
```
Comment (dsvanlani) [inline ui/src/styles/_colors.scss:32]:
```suggestion
```

**Review by dsvanlani — CHANGES_REQUESTED** [review]
Looks great, just some small things.

File: ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx (hunk 2 above)
Comment (shazlehu) [reply ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx]: This variable isn't available to the tool bar.

Comment (shazlehu) [reply ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx:43]: I'm also not seeing this error. I've found it's typically caused when you map an array to DOM elements, which we aren't doing in this case.

Comment (dsvanlani) [reply ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx:43]: Ooops, I meant you have to put it on the `<Stack>` - which we're mapping the TELEMETRY_TYPES to

**Review by dsvanlani — APPROVED** [review]
Just the one key prop issue, but looks good to me 👍 .

File: ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx (hunk 2 above)
Comment (shazlehu) [reply ui/src/components/MeasurementControlBar/MeasurementControlBar.tsx:43]: Git it, fixed.

**Review by andykellr — APPROVED** [review]

**Pull request merged**